packages/replay-next/src/suspense/BuildIdCache.ts

```diff
@@ -106,6 +149,13 @@ export const recordingCapabilitiesCache = createSingleEntryCache<
   debugLabel: "recordingCapabilitiesCache",
   load: async ([replayClient]) => {
     const recordingTarget = await recordingTargetCache.readAsync(replayClient);
-    return getRecordingCapabilities(recordingTarget);
+    const buildId = await buildIdCache.readAsync(replayClient);
+
+    const buildComponents = parseBuildIdComponents(buildId) ?? {
+      platform: recordingTarget,
+      runtime: recordingTarget,
+      date: "",
+    };
```

Comment:
nit but this feels more honest / less likely to possibly trigger any kind of false positive
```suggestion
    const buildComponents = parseBuildIdComponents(buildId) ?? {
      platform: "unknown",
      runtime: "unknown",
      date: "",
    };
```